sys/fs/cd9660/cd9660_node.c

```diff
@@ -146,8 +146,22 @@ cd9660_defattr(struct iso_directory_record *isodir, struct iso_node *inop,
 	}
 	if (!ap) {
 		inop->inode.iso_mode |= S_IRUSR|S_IXUSR|S_IRGRP|S_IXGRP|S_IROTH|S_IXOTH;
-		inop->inode.iso_uid = (uid_t)0;
-		inop->inode.iso_gid = (gid_t)0;
+		/*
+		 * Set some "sane" permissions for non-Rockridge volumes
+		 */
+		switch (inop->inode.iso_mode & S_IFMT) {
+		case S_IFREG:
+			/* Turn off executable bits for regular files */
+			inop->inode.iso_mode &= ~(VEXEC | VEXEC >> 3 | VEXEC >> 6);
+			/* FALLTHROUGH */
+		case S_IFDIR:
+			/* Turn on write permissions (owner) */
+			inop->inode.iso_mode |= VWRITE;
```

Comment:
Should be `S_IWUSR` instead of `VWRITE`, but also cd9660 in FreeBSD doesn't support read/write mounts so I'm not sure why it makes sense to add this permission for regular files and directories?

Reply:
Good catch.  I think I enabled this because when copying with the default `cp` alias (`alias cp='cp -ip'`) I ended up with non-writable files.  Will fix.

Reply:
Fixed.